front_end_ang/src/app/altersenha/altersenha.component.html

```diff
@@ -1,58 +1,72 @@
-
 <div class="col-md-6 mx-auto">
-
-  <h3 class="text-center">Alteração da senha do usuário</h3>
-
-  <form [formGroup]="ConfigForm" (ngSubmit)="onSubmit()">
-    <div class="col-md-4 mx-auto">
+    <h3 class="text-center" id="password-change-title">Alteração da senha do usuário</h3>
+  
+    <form [formGroup]="ConfigForm" (ngSubmit)="onSubmit()" aria-labelledby="password-change-title">
+      <div class="col-md-4 mx-auto">
+        <!-- Campo Senha Atual -->
         <div class="form-group">
-        <label for="oldPwd">Senha atual</label>
-        <input
+          <label for="oldPwd" id="oldPwd-label">Senha atual</label>
+          <input
             formControlName="oldPwd"
             id="oldPwd"
             type="password" 
-            class="form-control" />
-        <div class="alert alert-danger" *ngIf="oldPwd?.touched && oldPwd?.invalid">
-            <div *ngIf="oldPwd?.errors?.['required']">obrigatório</div>
+            class="form-control"
+            aria-labelledby="oldPwd-label"
```

Comment:
Desnecessário uso de aria-labelledby num input q já existe um label for referenciando